tests/loops/test_loops.py

```diff
@@ -756,7 +757,7 @@ def test_fit_loop_reset(tmpdir):
     ["train_datasets", "val_datasets"],
     [([RandomDataset], [RandomDataset]), ([RandomDataset], [RandomDataset, RandomDataset])],
 )
-@pytest.mark.parametrize("val_check_interval", [0.5, 1.0])
+@pytest.mark.parametrize("val_check_interval", [0.5])
```

Comment:
is 1.0 problematic?

Reply:
Good catch ! No, 0.5 does. Forgot to revert it back.